app/controllers/spots_controller.rb

```diff
@@ -0,0 +1,29 @@
+class SpotsController < ApplicationController
+  def index
+    @spots = Spot.all
+  end
+
+  def new
+    @spot = Spot.new
+  end
+
+  def create
+    @spot = Spot.new(spot_params)
+    if @spot.save
+      flash[:notice] = 'Spot added.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.